templates/kotlin/src/main/kotlin/io/appwrite/Client.kt.twig

```diff
@@ -62,11 +62,13 @@ class Client @JvmOverloads constructor(
     init {
         headers = mutableMapOf(
             "content-type" to "application/json",
+            "user-agent" to "{{spec.title | caseUcfirst}}{{ language.name | caseUcfirst }}SDK/{{ sdk.version }} ${System.getProperty("http.agent")}",
```

Comment:
Should `sdk.version` be in this?

Reply:
I think it was discussed and agreed to keep